### ISSUE TITLE
Multiple linked widgets do not render, error in JS console

### ISSUE THREAD

**Issue by parente** [jupyter-incubator/dashboards_nodejs_app#85]
Bottom left panel in https://github.com/jupyter-incubator/dashboards/blob/master/etc/notebooks/test/test_layout_basic.ipynb should have 3 linked widgets. Only one of them seems to render.  The other two generate "Could not create view" errors in the JS console

```
Error: Could not create view↵    at new Error (native)↵    at Error.WrappedError (http://127.0.0.1:3000/components/jupyter-js-widgets.js:31415:26)↵    at promiseRejection (http://127.0.0.1:3000/components/jupyter-js-widgets.js:31493:34
```

<img width="1147" alt="screen shot 2016-02-09 at 11 09 16 pm" src="https://cloud.githubusercontent.com/assets/153745/12938497/87162978-cf82-11e5-8810-4984397229c0.png">

**Comment by parente** [comment]
The `delete this._pendingExecutions[msg.parent_header.msg_id];` line is causing the widgets to go missing. Removing it allows additional widgets associated with the execute to appear. But when they do, they're not v-stacked properly like in the notebook. Probably styling at this point.

<img width="529" alt="screen shot 2016-02-09 at 11 23 08 pm" src="https://cloud.githubusercontent.com/assets/153745/12938619/3b492700-cf84-11e5-886c-88d42a9d8eb2.png">

**Comment by jhpedemonte** [comment]
Closed by #86.
